decayAmplitude/ampIntegralMatrixMetadata.cc

```diff
@@ -228,48 +222,12 @@ bool rpwa::ampIntegralMatrixMetadata::setAllZeroHash() {
 }
 
 
-bool rpwa::ampIntegralMatrixMetadata::addEventMetadata(const rpwa::eventMetadata& evtMeta, size_t eventMin, size_t eventMax) {
-	if (eventMin >= eventMax) {
-		printErr << "given event number bounds are not ordered correctly." << endl;
+bool rpwa::ampIntegralMatrixMetadata::addEventMetadata(const rpwa::eventMetadata& evtMeta) {
+	if (std::find(_evtMetas.begin(), _evtMetas.end(), evtMeta) != _evtMetas.end()) {
+		printWarn << "event metadata object has already been added." << endl;
```

Comment:
this is too prohibitive, the same event metadata might appear several times if the binning of integrals and events does not match